ui/tests/acceptance/secrets/backend/kv/secret-test.js

```diff
@@ -68,6 +68,8 @@ module('Acceptance | secrets/secret/create, read, delete', function (hooks) {
       assert
         .dom(PAGE.infoRowValue('Automate secret deletion'))
         .hasText('1 second', 'displays the delete version after set when configuring the secret-engine');
+      // [BANDAID] avoid error from missing param for links in SecretEdit > KeyValueHeader
+      await visit('/vault/secrets');
```

Comment:
ah darn, are adding those missing params something we can add easily in a different PR, or does it require a bigger change?

Reply:
KeyValueHeader already has a lot of complex logic, so it's not a simple fix. Ideally we want to replace that with `Page::Breadcrumbs` anyway, so it didn't feel worth the effort